jsonargparse_tests/test_typehints.py

```diff
@@ -489,6 +492,42 @@ def test_mapping_nested_without_args(parser):
     assert {"b": {"c": 2}} == parser.parse_args(['--map={"b": {"c": 2}}']).map
 
 
+def test_typeddict_without_arg(parser):
+    if sys.version_info < (3, 8):
+        pytest.skip("TypedDict introduced in python 3.8")
```

Comment:
Be consistent with the rest of the code. In other tests, skips are done using `@pytest.mark.skipif`, so please do the same in the added tests. See for example: https://github.com/omni-us/jsonargparse/blob/af37f088f257a184f1f9e77812539d8a2e584276/jsonargparse_tests/test_typehints.py#L525